pkg/lang/frontend/starlark/data/util.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2022 The envd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package data
+
+import (
+	"go.starlark.net/starlark"
+
+	envdData "github.com/tensorchord/envd/pkg/data"
```

Comment:
Camel case does not follow golang package convention, please use envddata or data